src/javascript/esutil.ts

```diff
@@ -229,9 +252,9 @@ export function toScannedMethod(
     privacy: getOrInferPrivacy(name, parsedJsdoc)
   };
 
-  const value = node.value;
-  if (value.type === 'FunctionExpression' ||
-      value.type === 'ArrowFunctionExpression') {
+  if (value &&
+      (babel.isFunctionExpression(value) ||
```

Comment:
If babel doesn't have a more general function predicate, we might want to make one so we have one canonical check and don't miss a node type.